README.md

```diff
@@ -700,8 +700,8 @@ profiles:
 ### RemoveFailedPods
 
 This strategy evicts pods that are in failed status phase.
-You can provide an optional parameter to filter by failed `reasons`.
-`reasons` can be expanded to include reasons of InitContainers as well by setting the optional parameter `includingInitContainers` to `true`.
+You can provide an optional parameter to filter by failed `reasons` and `exitCodes`.
+`reasons` and `exitCodes` can be expanded to include reasons of InitContainers as well by setting the optional parameter `includingInitContainers` to `true`.
```

Comment:
Worth mentioning the exit codes apply only when the pod's phase is failed.
> `reasons` and `exitCodes` can be expanded to include reasons of InitContainers as well by setting the optional parameter `includingInitContainers` to `true`.

Better to split the sentence into two. "can be expanded to include reasons of InitContainers" applies for `reasons` only.

Reply:
> You can provide an optional parameter to filter by failed `reasons` and `exitCodes`.

You can provide an optional parameter to filter by failed `reasons` or `exitCodes` or both.

Reply:
Changed the description to 

> Worth mentioning the exit codes apply only when the pod's phase is failed.
> 
> >

This strategy applies to Failed pod only IIUIC. I've changed the description to 
"You can provide optional parameters to filter by failed pods' `reasons`. and/or `exitCodes`. `exitCodes` apply to pods with `terminated` state only."

Reply:
> Worth mentioning the exit codes apply only when the pod's phase is failed.
> 
> > `reasons` and `exitCodes` can be expanded to include reasons of InitContainers as well by setting the optional parameter `includingInitContainers` to `true`.
> 
> Better to split the sentence into two. "can be expanded to include reasons of InitContainers" applies for `reasons` only.
`exitCode` applies to initContnainer too. This is actually the main use cases of the PR.

```
 kubectl explain pod.status.initContainerStatuses.state.terminated
KIND:       Pod
VERSION:    v1

FIELD: terminated <ContainerStateTerminated>

DESCRIPTION:
    Details about a terminated container
    ContainerStateTerminated is a terminated state of a container.

FIELDS:
  containerID	<string>
    Container's ID in the format '<type>://<container_id>'

  exitCode	<integer> -required-
    Exit status from the last termination of the container

  finishedAt	<string>
    Time at which the container last terminated

  message	<string>
    Message regarding the last termination of the container

  reason	<string>
    (brief) reason from the last termination of the container

  signal	<integer>
    Signal from the last termination of the container

  startedAt	<string>
    Time at which previous execution of the container started
    ```